website/static/js/profile.js

```diff
@@ -468,19 +479,37 @@ var SocialViewModel = function(urls, modes) {
     var self = this;
     BaseViewModel.call(self, urls, modes);
     TrackedMixin.call(self);
-
+    var i;
     self.addons = ko.observableArray();
 
-    self.personal = extendLink(
-        // Note: Apply extenders in reverse order so that `ensureHttp` is
-        // applied before `url`.
-        ko.observable().extend({
-            trimmed: true,
-            url: true,
-            ensureHttp: true
-        }),
-        self, 'personal'
-    );
+    self.canEditWebsite = ko.observable(false);
+
+    self.editWebsiteButton =  function(){
+        self.canEditWebsite(!self.canEditWebsite());
+    }
+
+    self.profileWebsites = ko.observableArray();
+                
+    self.hasProfileWebsites = ko.computed(function() {
+        if (self.profileWebsites())      
+        for (var i=0; i<self.profileWebsites().length; i++) {
+            if (self.profileWebsites()[i]) {
+                return true;
+            }
+        }
+        return false;
+    });
+    
+    self.canRemove = ko.computed(function () {
+        if (self.profileWebsites()) {
+            return self.profileWebsites().length > 1;
```

Comment:
Why should a user be required to have at least 1 website? Why can't they have 0?